source/guides/stacks/dlrs-inference.rst

```diff
@@ -0,0 +1,1035 @@
+.. _dlrs-inference:
+
+AI Inference with the Deep Learning Reference Stack
+###################################################
+
+In this guide we explore a solution for using the Deep Learning Reference Stack with a Seldon Core\* platform that has been deployed on Kubernetes\*.
+
+.. contents::
+   :local:
+   :depth: 2
```

Comment:
Could you please reset depth to 1? There's so much real estate taken up by the extra level, plus the overview.

Reply:
yep, done :)